### PR TITLE
#5 Do a virus scan at the start of the flow

### PR DESCRIPTION
**Pull request by janvanmansum**
@GeorgiKhomeriki please have look.

### CODE REVIEW

File: src/main/scala/nl/knaw/dans/easy/ingest_flow/EasyIngestFlow.scala [hunk 1]
```diff
@@ -100,6 +105,16 @@ object EasyIngestFlow {
     } yield datasetPid
   }
 
+  def assertNoVirusesInDeposit()(implicit s: Settings): Try[Unit] = Try {
+    import scala.sys.process._
```
Comment (GeorgiKhomeriki) [inline src/main/scala/nl/knaw/dans/easy/ingest_flow/EasyIngestFlow.scala]: Why the import here?

Comment (janvanmansum) [inline src/main/scala/nl/knaw/dans/easy/ingest_flow/EasyIngestFlow.scala]: No good reason, I can move it up.

**Comment by GeorgiKhomeriki** [comment]
:+1:

**Comment by janvanmansum** [comment]
Hm, strange, when I move it up, I am getting compiler errors in `queryPids`. I'll merge it like this for now. We can revisit it later.

**Comment by janvanmansum** [comment]
closes #14

**Pull request merged**